k8s/helm-charts/seldon-core-v2-setup/templates/seldon-v2-components.yaml

```diff
@@ -385,6 +385,30 @@ metadata:
   name: seldon-manager-config
   namespace: '{{ .Release.Namespace }}'
 ---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mlserver-models
+  namespace: '{{ .Release.Namespace }}'
+spec:
+  accessModes:
+  - ReadWriteOnce
```

Comment:
This allows reads/writes to the underlying PV **from a single k8s node**. This means things will start failing if the server replicas are spread across multiple nodes (as they should be in a HA setup). The problem is that the correct access mode (ReadWriteMany) is not directly supported by many cloud providers (GKE for example doesn't support it). Therefore we may need to think of a different solution for this (or storage of Model artifacts in Core 2 general)